gearbox/commands/serve.py

```diff
@@ -705,7 +705,10 @@ class GearboxWSGIServer(server_class): pass
 # For paste.deploy server instantiation (egg:gearbox#gevent)
 def gevent_server_factory(global_config, **kw):
     from gevent import reinit
-    from gevent.wsgi import WSGIServer
+    try:
+        from gevent.wsgi import WSGIServer
+    except ModuleNotFoundError:
+        from gevent.pywsgi import WSGIServer
```

Comment:
It's probably fine to reverse this, as `pywsgi` is the default one and `wsgi` was the exception.
Also as everything in TG has to support Python2.7, 3.4 and 3.5 `ModuleNotFound` error can't be used.